src/test/java/seedu/address/logic/parser/AddTaskCommandParserTest.java

```diff
@@ -77,6 +79,19 @@ public void parseCompulsoryFieldMissingFailure() {
                 missingBothErrorMessage() + "\n" + AddTaskCommand.MESSAGE_USAGE));
     }
 
+    @Test
+    public void parseInvalidFieldsPresentFailure() throws Exception {
+
+        //Task name shorter than 3 characters
+        assertParseFailure(parser, " " + PREFIX_TASKNAME + " " + INVALID_TASK_NAME_SHORT + DATETIME_DESC_TASKA + " "
+                + PREFIX_TAG + VALID_TAG_CHORES, Task.NAME_LENGTH_ERROR);
+
+        //Task name longer than 3 characters
```

Comment:
Perhaps the comment on this should be : //Task name longer than 50 characters instead?